specs/011-add-missing-module-specs/quickstart.md

```
@@ -0,0 +1,148 @@
+# Run the check again (expect it to fail)
+make check-specs
+# Expected exit code: 1
+
+# Restore
+cp tunnel/specs.md.bak tunnel/specs.md
+```
+
+**Expected outcome**: 
+
+For Test 1 (zero-byte), check fails with:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)